nbs/core.ipynb

```diff
@@ -989,6 +1014,29 @@
     "        if isinstance(fcsts_df, pd.DataFrame) and _id_as_idx():\n",
     "            _warn_id_as_idx()\n",
     "            fcsts_df = fcsts_df.set_index(self.id_col)\n",
+    "\n",
+    "        # perform conformal predictions\n",
+    "        if conformal_level is not None:\n",
+    "            if self._cs_df is None or self.conformal_intervals is None:\n",
+    "                warn_msg = (\n",
+    "                    'Please rerun the `fit` method passing a valid conformal_interval settings to compute conformity scores'\n",
+    "                )\n",
+    "                warnings.warn(warn_msg, UserWarning)\n",
```

Comment:
I think we should raise an Exception here - if a user passes a level but it's not possible compute that level an Exception feels more appropriate.